SPECS/fluent-bit/fluent-bit.spec

```diff
@@ -81,12 +80,19 @@ Development files for %{name}
 %{_unitdir}/fluent-bit.service
 %{_bindir}/*
 %{_prefix}%{_sysconfdir}/fluent-bit/*
+%exclude %{_bindir}/luajit
```

Comment:
Why would we remove the luajit binary if we are building the internal copy of luajit and turning on support for it?  (See previous comment)

Reply:
discussed offline.  External luajit support not available in this version of fluent-bit.